float32 interval
bool start
string file_name